src/plugins/auto/src/auto_compiled_model.cpp

```diff
@@ -241,7 +241,8 @@ ov::Any AutoCompiledModel::get_property(const std::string& name) const {
     } else if (name == ov::model_name) {
         std::lock_guard<std::mutex> lock(m_context->m_mutex);
         {
-            if (m_scheduler->m_compile_context[CPU].m_is_enabled && m_scheduler->m_compile_context[CPU].m_is_already)
+            if (m_scheduler->m_compile_context[CPU].m_is_enabled && m_scheduler->m_compile_context[CPU].m_is_already &&
+                m_scheduler->m_compile_context[CPU].m_compiled_model._ptr)
```

Comment:
likely ptr is not needed to check here because in the release function you reset both already and ptr?

Reply:
Yes, will update soon.